eclass/xdg.eclass

```diff
@@ -35,40 +35,49 @@ fi
 xdg_src_prepare() {
 	xdg_environment_reset
 
-	has ${EAPI:-0} 6 && default
+	[[ ${EAPI:-0} != [012345] ]] && default
```

Comment:
The check does not need to cover EAPI 0 to  3 at the very least. It would be interesting to check if any ebuild in tree uses EAPI 4 and 5 with this eclass as they are being deprecated and we can maybe drop them already from this eclass.

Reply:
True, I could adjust this check to make it more specific/explicit to supported/used
EAPIs.

Reply:
@EvaSDK also, wrt DEPEND I mostly just move stuff around in this series,
and none of the impacted eclasses have that depend to begin with.

Reply:
We shouldn't DEPEND on dev-util/gtk-update-icon-cache. Consumers of it should RDEPEND on it instead. gtk+ already does; up for Qt appropriate package to do it (and do an initial generation of it with its own postinst call to the eclass function). There's no point in the cache generation suceeding when there's nothing that will consume it - it just alienates people from using the eclass, especially for use cases where the GUI parts are optional behind a USE flag.

This is just a comment about this specific DEPEND - I will be able to take the time to review this all properly and add other changes on top of it at some point this January; probably not before 14th.

Reply:
Albeit that means people can't unmerge gtk-update-icon-cache (as it already is the case via gtk+ RDEP); I think the people that want to do that cleanly should be the people that implement recursive build depends :)